.github/BOTMETA.yml

```diff
@@ -27,7 +27,8 @@
 #           ignored - these people should never be notified
 #           deprecated - this file is deprecated but probably not yet renamed
 #           keywords - used to identify this file based on the issue description
-#           support - used for files without internal metadata
+#           support - used for files without internal ANSIBLE_METADATA, see
+#                     https://github.com/ansible/ansible/labels?q=support for full list
```

Comment:
i would argue it also is used for files WITH it i.e modules

Reply:
> I'm not sure if the bot combines support from ANSIBLE_METADATA and BOTMETA.yml.

Should a file be able to be supported both by `community` and `core` ? (response should be part of the [list of rules being defined](https://github.com/ansible/ansibullbot/pull/1008#pullrequestreview-150104847)) I mean is that `combine` or `override` ?

It seems that Ansibullbot is able to do that - using the `supported_by` keyword, not `support` -  and despite `supported_by` was never used in `BOTMETA.yml`:

1. `supported_by` `BOTMETA` has higher precedence than `supported_by` `ANSIBLE_METADATA` ([see](https://github.com/ansible/ansibullbot/blob/6ab590c39278471986a73d11d13fd6114fdb99f3/ansibullbot/utils/component_tools.py#L218-L219)).
2. when  `supported_by` is set,  `meta[support]` is equal to this variable ([see](https://github.com/ansible/ansibullbot/blob/6ab590c39278471986a73d11d13fd6114fdb99f3/ansibullbot/utils/component_tools.py#L1126))

Reply:
* My understanding is that a file must be supported by exactly ONE entity
* I think `ANSIBLE_METADATA` should override `BOTMETA`. This is because `ANSIBLE_METADATA` is used by the docsite
* AFAIK only Ansible Modules currently have a way of setting `support` via `ANSIBLE_METADATA`, it's possible in the future this may extend to other plugins.

Reply:
I wonder if this can be done by ignoring `BOTMETA` `support` for modules, therefore ONLY using what's in `ANSIBLE_METADATA`

Reply:
@gundalow that is a team decision to make. Create a bot issue and add it to a core team agenda or whatever it takes to drive a decision/consensus.

Reply:
We currently don't set `support` for anything under `lib/ansible/modules/` in BOTMETA

So I think the discussion here is about future proofing, and possibly adding ANSIBLE_METADATA to other plugin types (which would allow docs to show support status)

So I think this discussion is about future proofing/adding ANSIBLE_METADATA to other plugins

Therefore what's current in BOTMETA and Ansibulbot meta is correct